src/sentry/nodestore/django/backend.py

```diff
@@ -30,10 +30,13 @@ def get(self, id):
             return None
 
     def get_multi(self, id_list):
-        return dict(
-            (n.id, n.data)
+        return {
+            n.id: n.data
             for n in Node.objects.filter(id__in=id_list)
-        )
+        }
+
+    def delete_multi(self, id_list):
```

Comment:
This was just added since we don't have a `delete_multi` for django backend which would end up being `O(n)` over `DELETE` queries for each row individually.